src/playground/with-layout/chat/chat-custom-message-table.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'nb-custom-message-table',
+  template: `
+    <table>
+        <tr>
+            <th>{{tableHeader?.column1}}</th>
+            <th>{{tableHeader?.column2}}</th>
+            <th>{{tableHeader?.column3}}</th>
+        </tr>
+        <tr>
+            <td>Robert</td>
+            <td>Baratheon</td>
+            <td>46</td>
+        </tr>
+        <tr>
+            <td>Jaime</td>
+            <td>Lannister</td>
+            <td>31</td>
+        </tr>
+    </table>
+  `,
+})
+export class ChatCustomMessageTableComponent {
+  @Input() tableHeader: any;
+}
```

Comment:
Move this file into `components` subdirectory, so out playground schematics won't generate route for this component as we don't need it anyway.